clusterDeployer.py

```diff
@@ -136,6 +140,27 @@ def setup_all_vms(h: host.Host, vms: List[NodeConfig], iso_path: str) -> List[Fu
     return futures
 
 
+def deploy_microshift(cluster_name: str, node: NodeConfig, external_port: str, version: str) -> None:
```

Comment:
Now we can discuss where we can place this. It seems this function can be moved into the microshfit module.

Now you have functions like:

`deploy_microshift`
`microshift_iso_builder`

Step one: rename them so that they follow a uniform scheme:

`microshift_deploy`
`microshift_build_iso`

Step two: since they both live in the microshift module, you *could* (we can discuss what your thought is), remove `microshift_` from the name since it's implied by the fact that it's in that module. In this file, you can write `import microshift` and then use: `microshift.deploy(...`.

Reply:
sounds good!